### ISSUE TITLE
[Support]: Operation fails if Azure project contains spaces

### ISSUE THREAD

**Issue by stoe** [valet-customers/issue-ops#3]
## Description

Operation fails if Azure project contains spaces.

`gh valet` latest version via https://github.com/valet-customers/issue-ops
Running on `ubuntu-latest`

## How to reproduce

1. Open new issue via `Azure DevOps` template and provided the following information:
   - Organization: example-org
   - Project: My Demo Project
2. Add comment `/audit`
3. Action errors out with the message: `ERROR: "valet audit azure_devops" was called with arguments ["Demo", "Project"]`

## Relevant log output

```bash
gh valet audit azure-devops \
    --azure-devops-organization example-org --azure-devops-project "My Demo Project" \
    --output-dir output
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_INSTANCE_URL: https://github.com
    GITHUB_ACCESS_TOKEN: ***
    AZURE_DEVOPS_ACCESS_TOKEN: ***
ERROR: "valet audit azure_devops" was called with arguments ["Demo", "Project"]
Usage: "valet audit azure-devops"
```

## Workaround

Replace project name spaces with `%20`

1. Open new issue via `Azure DevOps` template and provided the following information:
   - Organization: example-org
   - Project: My%20Demo%2Project
2. Add comment `/audit`
3. Action runs successfully

---

Please let me know if this should be opened in https://github.com/github/gh-valet instead.

**Comment by ethanis** [comment]
@stoe thanks for opening this! This has been addressed in [v0.1.11](https://github.com/github/gh-valet/releases/tag/v0.1.11) of `github/gh-valet` and will be used automatically by the IssueOps repository 😎

**Comment by stoe** [comment]
Awesome. 
Thanks @ethanis
